R/utilities-configuration-plan.R

```diff
@@ -117,12 +117,69 @@ getCurvePropertiesForTimeProfiles <- function(configurationPlanCurve,
 
   # Observed Data
   if (pathArray[2] %in% "ObservedData") {
-    observedData <- getObservedDataFromConfigurationPlan(pathArray[1], configurationPlan, logFolder)
+    observedResults <- getObservedDataFromConfigurationPlan(pathArray[1], configurationPlan, logFolder)
+    molWeight <- getMolWeightForObservedData(pathArray, configurationPlan, simulation)

```

Comment:
@pchelle Please add some comments with e.g. an example how such a path of observed data or of simulated data looks like in the configuration plan. Otherwise it's not easy to understand, what is `pathArray[1]`, `pathArray[2]`, etc.

Reply:
Actually it would be better to refactor and define some additional functions like:
isObservedData(pathArray)
observedDataId(pathArray)
etc.
In those functions I would comment how the pathArray looks like; here in the "main" function pathArray would not appear at all.

Reply:
Yes, I'll add both as comment of the code and in the documentation of the `getObservedDataFromConfigurationPlan` fonction